controllers/postController.js

```diff
@@ -0,0 +1,76 @@
+const Post = require('../models/Post');
+
+const deletePost = (req, res) => {
+  const { postId } = req.params;
+
+  Post.delete(postId)
+    .then((data) => res.json(data.rows))
+    .catch((err) => {
+      console.log(err);
+      res.status(500).send(err);
+    });
+};
+
+const createPost = (req, res) => {
+  const { content } = req.body;
+  const { communityId, userId } = req.params;
+
+  Post.create(communityId, userId, content)
+    .then((data) => res.json(data.rows))
+    .catch((err) => {
+      console.log(err);
+      res.status(500).json({ error: '500 Internal Server Error' });
+    });
+};
+
+const getCommunityPosts = (req, res) => {
+  const { communityId } = req.params;
+  
+  Post.getCommunityPosts(communityId)
+    .then((data) => res.json(data.rows))
+    .catch((err) => {
+      console.log(err);
+      res.status(500).send(err);
+    });
+};
+
+const viewUsersPosts = (req, res) => {
```

Comment:
you should also include the community id incase the person is in multiple communities